vertexai/snippets/src/test/java/vertexai/gemini/samples/ChatDiscussionTest.java

```diff
@@ -0,0 +1,33 @@
+package vertexai.gemini.samples;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import java.io.IOException;
+
+import static com.google.common.truth.Truth.assertThat;
+import static com.google.common.truth.Truth.assertWithMessage;
+
+@RunWith(JUnit4.class)
+public class ChatDiscussionTest {
+    private static final String GCP_PROJECT_VAR_NAME = "GOOGLE_CLOUD_PROJECT";
```

Comment:
```suggestion
    private static final String GOOGLE_CLOUD_PROJECT = System.getenv("GOOGLE_CLOUD_PROJECT");
```

Kindly follow this naming pattern.